src/routes/sidebar.js

```diff
@@ -0,0 +1,44 @@
+import { StackNavigator } from 'react-navigation';
+import navigationOptions from './navigationOptions';
+import SidebarScreen from '../screens/Sidebar/SidebarScreen';
```

Comment:
if you merge in the latest master we have an alias for the src folder now and you can change all your relative paths to start with `kitsu`

Good rule of thumb: if its' not in your current folder, start with `kitsu`